app/_features/event/components/event-detail.tsx

```diff
@@ -159,3 +155,124 @@ export default function EventDetail({
     </>
   );
 }
+
+function AuthorActionButtons({
+  handleCopyLink,
+  slug,
+  copiedActive,
+}: {
+  handleCopyLink: (text?: string) => Promise<void>;
+  slug: string;
+  copiedActive: boolean;
+}) {
+  const [isEditing, setIsEditing] = useState(false);
+
+  const openDeleteEventModal = useCallback(() => {
+    document.dispatchEvent(new CustomEvent('open:event-delete-modal'));
+  }, []);
+
+  const { navigateTo } = useNavigate();
+
+  const onEditEvent = useCallback(() => {
+    setIsEditing(true);
+    navigateTo(new URL(`/event/${slug}/edit`, window.location.origin).href);
+  }, [navigateTo, slug]);
+
+  return (
+    <div className="flex justify-between gap-2">
+      <Button
+        onClick={openDeleteEventModal}
+        className="tems-center flex aspect-[1/1] rounded-md bg-zinc-800 py-0 text-base font-medium text-zinc-100 antialiased hover:bg-zinc-700 active:bg-zinc-600"
+        variant="flat"
+        isIconOnly
+      >
+        <DeleteIcon width={20} height={20}></DeleteIcon>
+      </Button>
+      <div className="flex w-full gap-2">
+        <Button
+          onPress={onEditEvent}
+          variant="flat"
+          className="flex min-w-0 basis-1/2 items-center gap-1.5 rounded-md bg-zinc-800 text-base font-medium text-zinc-100 antialiased hover:bg-zinc-700 active:bg-zinc-600"
+        >
+          {isEditing ? (
+            <div className="flex gap-2">
+              <Spinner
+                classNames={{
+                  circle1: 'border-b-zinc-200',
+                  circle2: 'border-b-zinc-200',
+                  wrapper: 'w-4 h-4',
+                }}
+              />
+              <span>Editing...</span>
+            </div>
+          ) : (
+            <div className="flex items-center gap-unit-2 align-middle">
+              <EditIcon height={20} width={20} />
+              <span>Edit Event</span>
+            </div>
+          )}
+        </Button>
```

Comment:
Similar to [this comment](https://github.com/inlivedev/inlive-room/pull/142#discussion_r1478094200). This edit button is just a navigation link right? Please use `<Link>` nextjs component with the `<Button>` instead.